openedxstats/apps/sites/views.py

```diff
@@ -24,7 +24,8 @@
 
 # Converts site data into JSON format for Ajax request
 def SiteView_JSON(request):
-    sites = Site.objects.all()
+    all_site_geo_zone = SiteGeoZone.objects.prefetch_related()
```

Comment:
`all_site_geo_zone` is a new variable, but doesn't seem used by the changes.  Do we need it?